src/main/requests/page/user_long_lived_token/GetUserLongLivedTokenResponse.ts

```diff
@@ -0,0 +1,26 @@
+import { AbstractResponse } from '../../AbstractResponse';
+import { UserLongLivedTokenData } from '../../data/UserLongLivedTokenData';
+
+/**
+ * This class is used to exchange a short-lived-token by a long-lived-token that you get when
+ * the user do a Facebook login in your page and want to authorize your app
+ * to make request for a long time.
+ *
+ * This long-lived-token last ~60 days according to Facebook documentation
+ *
+ * More info:
+ * https://developers.facebook.com/docs/facebook-login/access-tokens/refreshing/#get-a-long-lived-user-access-token
+ *
```

Comment:
Change this to simply be "Represents a response from the {@link GetUserLongLivedTokenRequest}"